docs/modules/ROOT/pages/cops_rspec.adoc

```diff
@@ -1145,21 +1150,49 @@ around { |test| test.run }
 it { does_something }
```

Comment:
btw there is an extra space in the existing examples

Reply:
not sure to understand sorry